chart/kubeapps/values.yaml

```diff
@@ -202,6 +202,11 @@ frontend:
   ## Some K8s distributions such as GKE requires it
   ##
   proxypassAccessTokenAsBearer: false
+  ## Set large_client_header_buffers in nginx config
+  ## Can be required when using OIDC or LDAP due to large cookies
+  ## Example value: "4 32k"
+  #
+  largeClientHeaderBuffers: ""
```

Comment:
Thanks @rarick . I'd be +1 to make the default "4 32k", given that I'm keen to have the OIDC setup as the default one, though it's still good for it to be configurable (people can switch it off or make it bigger if, for eg., there setup has many groups being encoded into the OIDC id_token)

What do you think?

Reply:
That sounds good to me. In the scenario that broke our setup, group membership in the token pushed the cookie size to just over 8k, so 32k should be fine for the vast majority of users. I'll get that in right away 👍

Reply:
Yeah, I had a similar thing while testing OIDC + LDAP (#1944, but in that case it was on the ingress nginx for a dev setup). Excellent, thanks for the change!